pulsar-broker/src/main/java/org/apache/pulsar/broker/TransactionMetadataStoreService.java

```diff
@@ -271,10 +269,8 @@ public long getLowWaterMark(TxnID txnID) {
                         LOG.error("EndTransaction UpdateTxnStatus fail! TxnId : {}, "
                                 + "TxnAction : {}", txnID, txnAction, e);
                     } else {
-                        if (LOG.isDebugEnabled()) {
-                            LOG.debug("EndTransaction UpdateTxnStatus op retry! TxnId : {}, "
+                        LOG.debug("EndTransaction UpdateTxnStatus op retry! TxnId : {}, "
```

Comment:
These calls with multiple args would trigger the allocation of a `Object[]` to contain them.

Reply:
I fixed it.